tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -51,13 +51,7 @@
 # @result namespacestr [String] Returns 'sudo ip netns exec vrf <cmd>'
 # command string for 'cisco' platform.
 def get_namespace_cmd(host, cmdstr, options)
-  case host['platform']
-  when /cisco/
-    agentvrf = options[:HOSTS][host.to_s.to_sym]['vrf']
-    return "sudo ip netns exec #{agentvrf} " + cmdstr
-  else
-    return cmdstr
-  end
+  return cmdstr
```

Comment:
Your PR description says: "Beaker now supports getting VRF from host object."

Your change no longer references host. This method is supposed to return cmdstr with the appropriate namespace / vrf context.

So I don't quite understand. Can you explain how this is supposed to work in a vrf after your change?